src/util/util.ts

```diff
@@ -13,15 +13,6 @@ export function unique<T>(xs: T[]) {
     return [...new Set(xs)];
 }
 
-export async function someAsync<T>(arr: ReadonlyArray<T>, f: (t: T) => Promise<boolean>): Promise<boolean> {
```

Comment:
Leftover since https://github.com/DefinitelyTyped/dt-mergebot/pull/214/files#diff-34e181228489bdec8b91811ab9feec5648a50dfbc514704a1ed983ceaac61bf5L69?